unlock-js/src/__tests__/helpers/nockHelper.js

```diff
@@ -2,19 +2,34 @@
 import nock from 'nock'
 
 export class NockHelper {
-  constructor(endpoint, debug = false) {
+  constructor(endpoint, debug = false, isEthers = false) {
     this.nockScope = nock(endpoint, { encodedQueryParams: true })
 
+    this.anyRequestSetUp = false
     this.debug = debug
-    this._rpcRequestId = 0
+    this._rpcRequestId = isEthers ? 42 : 0
     this._noMatches = []
+    this._isEthers = isEthers
 
     // In order to monitor traffic without intercepting it (so that mocks can be built). uncomment the line below
     // nock.recorder.rec()
 
     nock.emitter.on('no match', (clientRequestObject, options, body) => {
       this._noMatches.push(body)
       if (debug) {
+        if (!this.anyRequestSetUp) {
+          console.log(
+            new Error('No mocks have been set up, but a request was made!')
+          )
+        }
+        if (!body) {
+          console.log(
+            new Error(
+              'no body? (is there a jest.fakeTimers call? you must reset mocks' +
```

Comment:
Good catch there, that'll certainly save some hair pulling in the future

Reply:
you should have seen my scalp last week..